OpenSim/Simulation/SimbodyEngine/SimbodyEngine.h

```diff
@@ -120,8 +120,26 @@ OpenSim_DECLARE_CONCRETE_OBJECT(SimbodyEngine, Object);
     //--------------------------------------------------------------------------
     // CONSTRAINTS
     //--------------------------------------------------------------------------
-    virtual void formCompleteStorages( const SimTK::State& s, const OpenSim::Storage &aQIn,
-       OpenSim::Storage *&rQComplete,OpenSim::Storage *&rUComplete) const;
+    /**
+    * From a potentially partial specification of the generalized coordinates,
+    * form a complete storage of the generalized coordinates (q's) and
+    * generalized speeds (u's) in radians and radians/s respectively.
+    *
+    * @param aQIn Storage containing the q's or a subset of the q's.
+    * @param rQComplete Storage containing all the q's.  If q's were not
+    * in aQIn, the values are set to 0.0.  When a q is constrained, its value
+    * is altered to be consistent with the constraint.  The caller is responsible
+    * for deleting the memory associated with this storage. Units are radians.
+    * @param rUComplete Storage containing all the u's.  The generalized speeds
+    * are obtained by spline fitting the q's and differentiating the splines.
+    * When a u is constrained, its value is altered to be consistent with the
+    * constraint.  The caller is responsible for deleting the memory
+    * associated with this storage. Units are radians/s.
+    */
```

Comment:
Can we explicitly state which storages are in radians, if all then just state that. Did we search all the codebase to make sure the change in the implementation of the functionality doesn't break anything else.

Reply:
It explicitly states that the outputs, rQComplete and r UComplete are in radians and radians/s.  The input doesn't matter as long as the `Storage` is `inDegrees` if contains angles in degrees. That has not changed. The only change was to the output Storages which were always returning degrees.